markdown/events/2022/bytesize-proteinfold.md

```diff
@@ -0,0 +1,23 @@
+---
+title: 'Bytesize: nf-core/proteinfold'
+subtitle: Athanasios Baltzis - Center for Genomic Regulation, Barcelona 
```

Comment:
Just a small typo: Centre for Genomic Regulation

Reply:
Tomato / tomato 😀  (Center is American spelling)

```suggestion
subtitle: Athanasios Baltzis - Centre for Genomic Regulation, Barcelona 
```